tsconfig.json

```diff
@@ -13,7 +13,8 @@
 			"src/typings"
 		],
 		"sourceMap": true,
-		"rootDir": "src"
+		"rootDir": ".",
+		"resolveJsonModule": true
```

Comment:
Why did this have to be added?  I don't see it in our extensions.

Reply:
That's required for importing from a package.json here: https://github.com/microsoft/vscode-azure-account/blob/82554472a1866d3494b60e5ffeefd929a30c715d/src/telemetry.ts#L7

Typescript throws this error without it:
```
TS2732: Cannot find module '../package.json'. Consider using '--resolveJsonModule' to import module with '.json' extension
```

Reply:
So odd